include/deal.II/sundials/kinsol.h

```diff
@@ -517,9 +517,9 @@ namespace SUNDIALS
      * that this may not converge, or may converge very slowly.
      *
      * A call to this function should store in `dst` the result of $J^{-1}$
-     * applied to `rhs`, i.e., `J*dst = rhs`. It is the user's responsibility
-     * to set up proper solvers and preconditioners inside this function
-     * (or in the `setup_jacobian` callback above).
+     * applied to `rhs`, i.e., `J` $*$ `dst = rhs`. It is the user's
```

Comment:
Do you know how that would look like? You could also use \cdot:
``$ J \cdot dst = rhs$``